src/main/java/com/google/sps/servlets/RandomProfileServlet.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2019 Google LLC
```

Comment:
What is this file doing here?

Reply:
We are displaying the teammates in a random order.

Reply:
This is an overkill, just display them in alphabetical order!